packages/@react-aria/dnd/src/utils.ts

```diff
@@ -336,6 +336,13 @@ export function setGlobalDnDState(state: DnDState) {
   globalDndState = state;
 }
 
+// Util function to check if the current dragging collection ref is the same as the current targeted droppable collection ref.
+// Allows a droppable ref arg in case the global drop collection ref hasn't been set
+export function isInternalDropOperation(ref?: RefObject<HTMLElement>) {
```

Comment:
`isInternal`? `isInternalDrop`? `isInternalTarget`?

Reply:
I think what you have is good